eland/ml/pytorch/transformers.py

```diff
@@ -584,7 +584,7 @@ def __init__(
         self,
         *,
         model_id: str,
-        access_token: Optional[str],
+        access_token: Optional[str] = None,
```

Comment:
The order of `access_token` does not matter because those are kwargs-only parameter that can't be passed conditionally.

Reply:
Stylistically, I always prefer default args are at the end, but I don't know if this is more or less Pythonic. Either way is fine, just curious

Reply:
Ah, that's an interesting consideration. I don't know if there's one choice that's more Pythonic than the other (I haven't seen enough such cases to tell).

I don't have a strong preference so I am happy to change this.